reconstruction/dc/src/main/java/org/jlab/rec/dc/banks/HitReader.java

```diff
@@ -205,38 +205,24 @@ private int getJitter(int sector, int layer, int wire, int order) {
         }
         return jitter;
     }
-    
-    public void fetch_DCHits(DataEvent event, Clas12NoiseAnalysis noiseAnalysis,
-                             NoiseReductionParameters parameters,
-                             Clas12NoiseResult results) {
-        this.initialize(event);
-        this.fetch_DCHits(noiseAnalysis, parameters, results);
-    }
-        
+            
      /**
      * reads the hits using clas-io methods to get the EvioBank for the DC and
      * fill the values to instantiate the DChit and MChit classes.This methods
      * fills the DChit list of hits.
-     *
-     * @param noiseAnalysis
-     * @param parameters
-     * @param results
      */
-    private void fetch_DCHits(Clas12NoiseAnalysis noiseAnalysis,
-                             NoiseReductionParameters parameters,
-                             Clas12NoiseResult results) {
-
+    public void fetch_DCHits(DataEvent event) {
+        this.initialize(event);
+        
         _DCHits = new ArrayList<>();
-
-        IndexedList<Boolean> noise = new IndexedList<>(4);
         
         RawDataBank bankDGTZ = new RawDataBank(bankNames.getTdcBank(), OrderGroups.NODENOISE);
```

Comment:
This, up to line 239, is no more used and can be removed

Reply:
Done.